lib/gumboot/shared_examples/foreign_keys.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.shared_examples 'Foreign Keys' do
+  conn = ActiveRecord::Base.connection
+  context 'Permissions Foreign Keys', if: conn.supports_foreign_keys? do
```

Comment:
Can the `if` statement you have here, which is duplicated through the file be applied once at the `shared_examples` level by chance?